### PR TITLE
docs: fix spelling mistakes

### PR DESCRIPTION
**Pull request by hwdef**

### CODE REVIEW

File: docs/apireference/build/documents/_generated_cluster_v1alpha2_kops_concept.md [hunk 1]
```diff
@@ -113,7 +113,7 @@ Appears In:
 
 Field        | Description
 ------------ | -----------
-additionalNetworkCIDRs <br /> *string array*    | AdditionalNetworkCIDRs is a list of aditional CIDR used for the AWS VPC or otherwise allocated to k8s. This is a real CIDR, not the internal k8s network On AWS, it maps to any aditional CIDRs added to a VPC.
+additionalNetworkCIDRs <br /> *string array*    | AdditionalNetworkCIDRs is a list ofadditional CIDR used for the AWS VPC or otherwise allocated to k8s. This is a real CIDR, not the internal k8s network On AWS, it maps to anyadditional CIDRs added to a VPC.
```
Comment (bittopaz) [inline docs/apireference/build/documents/_generated_cluster_v1alpha2_kops_concept.md]: Maybe you missed space in the words.

File: docs/apireference/build/documents/_generated_instancegroup_v1alpha2_kops_concept.md [hunk 2]
```diff
@@ -77,7 +77,7 @@ Appears In:
 Field        | Description
 ------------ | -----------
 additionalSecurityGroups <br /> *string array*    | AdditionalSecurityGroups attaches additional security groups (e.g. i-123456)
-additionalUserData <br /> *[UserData](#userdata-v1alpha2-kops) array*    | AdditionalUserData is any aditional user-data to be passed to the host
+additionalUserData <br /> *[UserData](#userdata-v1alpha2-kops) array*    | AdditionalUserData is anyadditional user-data to be passed to the host
```
Comment (bittopaz) [inline docs/apireference/build/documents/_generated_instancegroup_v1alpha2_kops_concept.md]: A space is needed in "anyadditional"

File: docs/apireference/build/documents/_generated_cluster_v1alpha2_kops_concept.md (hunk 1 above)
Comment (hwdef) [reply docs/apireference/build/documents/_generated_cluster_v1alpha2_kops_concept.md]: Yes,it is my mistake.
I fixed it.

**Comment by justinsb** [comment]
Thanks @hwdef - some of these were originally generated files, but it looks like they may have become detached from their generation.  Thanks for fixing!

/approve
/lgtm

**Pull request merged**